stress/common/src/main/java/alluxio/stress/fuse/FuseIOSummary.java

```diff
@@ -77,46 +69,31 @@ public GraphGenerator graphGenerator() {
     return null;
   }
 
-  /**
-   * @return a map mapping job worker unique id to its throughput
-   */
-  public Map<String, Float> getClientsThroughput() {
-    return mClientsThroughput;
-  }
-
-  /**
-   * @param clientsThroughput the map mapping job worker unique id to its throughput
-   */
-  public void setClientsThroughput(Map<String, Float> clientsThroughput) {
-    mClientsThroughput = clientsThroughput;
-  }
-
```

Comment:
The clientThroughtput is now saved in mNodeResults, I think we don't need to save these throughtput data again as a parameter. So I removed the parameter and its getter and setter.

Reply:
@ssz1997 @jja725 FYI the interface has slightly changed but we didn't remove any information. They are just organized in different ways now.